Author: Ole Streicher <olebole@debian.org>
Description: Put the shared libraries into a package specific subdir
--- a/utility/binary/CMakeLists.txt
+++ b/utility/binary/CMakeLists.txt
@@ -50,7 +50,7 @@
 
     #! Set install locations for standalone mode.
     if (NOT OSKAR_LIB_INSTALL_DIR)
-        set(OSKAR_LIB_INSTALL_DIR "lib")
+        set(OSKAR_LIB_INSTALL_DIR "lib/oskar")
     endif()
     if (NOT OSKAR_INCLUDE_INSTALL_DIR)
         set(OSKAR_INCLUDE_INSTALL_DIR "include/oskar_binary")
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -26,7 +26,7 @@
 
 # === Setup install directory paths
 if (NOT OSKAR_LIB_INSTALL_DIR)
-    set(OSKAR_LIB_INSTALL_DIR "lib")
+    set(OSKAR_LIB_INSTALL_DIR "lib/oskar")
 endif()
 if (NOT OSKAR_INCLUDE_INSTALL_DIR)
     set(OSKAR_INCLUDE_INSTALL_DIR "include/oskar")
@@ -131,30 +131,6 @@
 else()
     set(install_include_dir "${CMAKE_INSTALL_PREFIX}/include")
 endif ()
-# TODO move install command into subdirectories to make this more modular,
-# and give subdirectories control over which headers are installed.
-install(DIRECTORY "${PROJECT_SOURCE_DIR}/."
-    DESTINATION ${OSKAR_INCLUDE_INSTALL_DIR}
-    FILES_MATCHING PATTERN "oskar_*.h"
-    PATTERN ${install_include_dir} EXCLUDE
-    PATTERN build EXCLUDE
-    PATTERN release EXCLUDE
-    PATTERN release* EXCLUDE
-    PATTERN debug EXCLUDE
-    PATTERN debug* EXCLUDE
-    PATTERN .settings EXCLUDE
-    PATTERN cmake EXCLUDE
-    PATTERN doc EXCLUDE
-    PATTERN gui EXCLUDE
-    PATTERN matlab EXCLUDE
-    PATTERN python EXCLUDE
-    PATTERN *dierckx* EXCLUDE
-    PATTERN src  EXCLUDE
-    PATTERN test EXCLUDE
-    PATTERN extern EXCLUDE
-    PATTERN .svn EXCLUDE)
-install(FILES ${PROJECT_BINARY_DIR}/oskar_version.h
-    DESTINATION ${OSKAR_INCLUDE_INSTALL_DIR})
 
 # === Copy cleanup bash script into the build directory
 configure_file(${PROJECT_SOURCE_DIR}/cmake/oskar_cleanup.sh
